kotlin-runtime/ftl-runtime/src/main/kotlin/xyz/block/ftl/schemaextractor/ExtractSchemaRule.kt

```diff
@@ -214,44 +214,55 @@ class SchemaExtractor(
       }
 
       // Validate parameters
-      require(verb.valueParameters.size == 2) { "$verbSourcePos Verbs must have exactly two arguments, ${verb.name} did not" }
+      require(verb.valueParameters.size >= 1) { "$verbSourcePos Verbs must have at least one argument, ${verb.name} did not" }
+      require(verb.valueParameters.size <= 2) { "$verbSourcePos Verbs must have at most two arguments, ${verb.name} did not" }
```

Comment:
@worstell would be good to get your 👀 on some of this `kotlin` stuff if you have a moment.

Reply:
lgtm!